pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -2236,6 +2236,57 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			)
 		})
 
+		Context("Descheduler annotations", func() {
+
+			It("should add eviction-in-progress annotation in case of VMI marked for eviction", func() {
+				vmi := newPendingVirtualMachine("testvmi")
+				vmi.Status.Phase = virtv1.Running
+				vmi.Status.EvacuationNodeName = "test"
+
+				pod := newPodForVirtualMachine(vmi, k8sv1.PodRunning)
+				Expect(pod.Annotations).ToNot(HaveKey(descheduler.EvictionInProgressAnnotation))
+				addActivePods(vmi, pod.UID, "")
+				addVirtualMachine(vmi)
+				addPod(pod)
+
+				sanityExecute()
+
+				updatedPod, err := kubeClient.CoreV1().Pods(pod.Namespace).Get(context.Background(), pod.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(updatedPod.Annotations).To(HaveKey(descheduler.EvictionInProgressAnnotation))
+
+				updatedVmi, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(vmi.IsMarkedForEviction()).To(BeTrue())
+				Expect(updatedVmi.Annotations).To(HaveKey(descheduler.EvictionInProgressAnnotation))
+			})
+
+			It("should remove eviction-in-progress annotation in case of VMI marked for eviction", func() {
```

Comment:
```suggestion
			It("should remove eviction-in-progress annotation in case of VMI not marked for eviction", func() {
```

It should not be marked, right?

Reply:
yep, thanks